stdlib/distutils/core.pyi

```diff
@@ -52,6 +52,7 @@ def setup(
     include_dirs: list[str] = ...,
     password: str = ...,
     fullname: str = ...,
-    **attrs: Any,
+    # Custom Distributions could accept more params
+    **attrs: object,
```

Comment:
`Any` seems correct here. The accepted types depend on what the custom distributions accept (which can't be expressed in the type system.) It doesn't accept anything without limitations.

```suggestion
    # Custom Distributions could accept more params
    **attrs: Any,
```

Reply:
Fair. Its just semantics at this point, so as long as it's commented anyway.